src/js/main.js

```diff
@@ -1,6 +1,78 @@
-import {animatedScrollTo} from './es6-scroll-to.js';
+console.log("ES6");
+const contactUs = {
 
-document.querySelector('.title').innerHTML = 'Great success!'
-document.querySelector('.text').innerHTML = 'Javascript is working!'
+  button: document.getElementsByClassName("imageText__button"),
+  ctaSection: document.querySelectorAll('.contactUs'),
```

Comment:
Here you're using single quotes, in other places double quotes. Please be consistent